src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -191,6 +192,7 @@ class ItemPage extends React.Component {
     const externalLinks = this.getDisplayFields(record, externalFields);
     const itemDetails = this.getDisplayFields(record, displayFields);
     const itemOverview = this.getDisplayFields(record, overviewFields);
+    const sortBy = '';
```

Comment:
On line 284 you have `sortBy` being pulled in from the component's props. Here you should have `const sortBy = this.props.sortBy;` to get the sortBy value.